src/index.html

```diff
@@ -2,12 +2,46 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+        src="images/imac.jpeg"
+        alt="imac_images"
+        class="card__img"
+        >
+      <h2 class="card__title">APPLE A1419 iMac 27" Retina <br> 5K Monoblock (MNED2UA/A)</h2>
+      <p class="card__id">Product code: 195434</p>
+
+      <div class="grade">
```

Comment:
this is not a separate BEM block but an element of card block. use proper class name

Reply:
<img width="654" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/70220667/db84e082-0cf4-4fea-8ac3-2acd40cc481e">